cves/CVE-2018-1336.yml

```diff
@@ -68,8 +70,8 @@ fixes:
    - commit:
      note:
 vccs:
-  - commit:
-    note:
+  - commit: ac4010b392a4406538ec5b29626f362c3e806c5b
+    note: SVN rev 1420654
```

Comment:
Would it be possible to give a bit more information in the note for this commit?